elasticsearch/run.sh

```diff
@@ -28,4 +39,24 @@ else
 	exit 1
 fi
 
-exec /usr/share/elasticsearch/bin/elasticsearch
+ln -s /usr/share/elasticsearch/config/${CLUSTER_NAME}.truststore /usr/share/elasticsearch/config/${CLUSTER_NAME}.truststore.jks
+cp /etc/elasticsearch/secret/admin-jks /usr/share/elasticsearch/config/admin.jks
+
+TIMES=60
+function waitForES() {
+  for (( i=1; i<=$TIMES; i++ )); do
+    # test for ES to be up first
+		# don't provide a client cert since ES doesn't validate us correctly
```

Comment:
SG doesn't poll its index anymore as it did in 1.5, it listens for a change on specific ES types.  We can seed correctly when we receive a REST request without this function; but in the scenario where we just start up an ES cluster with say 2 nodes, one may not initialize its ACL and then SG starts to complain that it isn't initialized...